testsuite/config/enable-elytron-full.cli

```diff
@@ -0,0 +1,25 @@
+embed-server --server-config=${jboss.server.config.file.name}
```

Comment:
I've not looked, but is authentication used in the Spring tests? If not we may not need this is all.

Reply:
okay! I guess we don't need it.